components/inputs/input-text.js

```diff
@@ -135,9 +135,9 @@ class InputText extends PropertyRequiredMixin(FocusMixin(LabelledMixin(FormEleme
 			 */
 			step: { type: String },
 			/**
-			 * Text for additional screenreader and mouseover context
-			 * @type {string}
+			 * @ignore
 			 */
+			// eslint-disable-next-line lit/no-native-attributes
```

Comment:
I would have liked to remove this like number/percent but there were just too many search results to comb through. For now it'll be hidden from the docs and maybe we do a more concerted search effort later.